tests/models/test_dagrun.py

```diff
@@ -31,15 +31,14 @@
 from airflow.decorators import setup, task, task_group, teardown
 from airflow.exceptions import AirflowException
 from airflow.models import (
-    DAG,
     DagBag,
```

Comment:
Here you missed some small pieces :-D

Reply:
Same comment, if you will update them in a separate PR, then LGTM